style.css

```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Poppins:wght@500&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.main_section {
+  background: url('images/header-shapes\ mobile.svg') no-repeat center center;
+  background-color: #fff;
+  max-width: 375px;
+  height: 100vh;
+}
+
+.material-icons {
+  color: #6070ff;
+  font-size: larger;
+  font-weight: bolder;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 9%;
+  margin-left: -20px;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  margin-left: -20px;
+}
+
+h1 {
+  margin-top: 58px;
+  font-size: 50px;
+  font-weight: bolder;
+  color: #2e394b;
+  text-align: justify;
+  line-height: normal;
+}
+
+.logo-left {
+  color: #6070ff;
+  font-size: larger;
+  font-weight: bolder;
+}
+
+.contacts,
+img {
+  margin-top: -8px;
+}
+
+.contacts {
+  margin-left: -20px;
+}
+
+h5 {
+  font-size: 10px;
+  margin-top: -6px;
+  color: #6070ff;
+}
```

Comment:
- [ ] [OPTIONAL] This element still doesn't look exactly like the design 👇 
![Screenshot from 2022-05-28 11-00-17](https://user-images.githubusercontent.com/87965534/170816476-35d7e4fd-4cdf-4407-9d41-5e5de9ea0e8a.png)
Figma design: 
![Screenshot from 2022-05-28 11-00-32](https://user-images.githubusercontent.com/87965534/170816491-4fb55839-08e2-4717-803b-eb9e5b99ac9a.png)
you can fix that by adding these properties: 
```
color: #7F8CFF;
font-weight: 500;
font-size: 16px;
```